### PR TITLE
meshroom: Add version 2021.1.0

### PR DESCRIPTION
**Pull request by r15ch13**
Add Meshroom version 2021.1.0

### CODE REVIEW

**Comment by issaclin32** [comment]
config location: `$Env:LocalAppData\AliceVision\Meshroom`
(This is just a reminder, no need to modify anything)

File: bucket/meshroom.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/alicevision/meshroom"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/alicevision/meshroom/releases/download/v$version/Meshroom-$version-win64.zip",
+                "extract_dir": "Meshroom-$version"
```
[the remark this answers is not in the document]
Comment (r15ch13) [reply bucket/meshroom.json:27]: The md5 hash was only provided on the last release. I will add this md5 hash to the manifest, but the checkver should still use a normal version regex.

**Comment by issaclin32** [comment]
> The md5 hash was only provided on the last release.


My thought is, since the md5 hash is provided on the last release, I would expect them to **provide it in future releases**.

Even if they don't:
**(1)** We will have not problem updating the package, because *Scoop* will download the file when hash extraction failed.

**(2)** Some of our maintainers, including me and @tech189 are regularly checking *Excavavor* logs to see if there is any **invalid urls** and **autoupdate failures**. So if this didn't work, we will just remove `autoupdate.hash` later.

**Review by tech189 — COMMENTED** [review]
Program installs fine, but I'm getting an empty command prompt window that closes the main GUI if I close it, is that normal?

**Comment by issaclin32** [comment]
> Program installs fine, but I'm getting an empty command prompt window that closes the main GUI if I close it, is that normal?

I think this is a pretty common case for GUI apps made with **Python**.
If we run a GUI app via `python.exe` (instead of `pythonw.exe`). Then, an empty command prompt window will appear.

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**